client/benches/tps/lib.rs

```diff
@@ -117,15 +123,39 @@ impl MeasurerUnit {
     /// Submit initial transactions for measurement
     #[allow(clippy::expect_used, clippy::unwrap_in_result)]
     fn ready(self) -> Result<Self> {
-        let (public_key, _) = iroha_core::prelude::KeyPair::generate()
-            .expect("Failed to generate KeyPair.")
-            .into();
-
-        let register_me = RegisterBox::new(Account::new(account_id(self.name), [public_key]));
-        let mint_a_rose = MintBox::new(1_u32, asset_id(self.name));
+        let keypair =
+            iroha_core::prelude::KeyPair::generate().expect("Failed to generate KeyPair.");
+
+        let account_id = account_id(self.name);
+        let alice_id = <Account as Identifiable>::Id::from_str("alice@wonderland")?;
+        let asset_id = asset_id(self.name);
+
+        let register_me = RegisterBox::new(Account::new(
+            account_id.clone(),
+            [keypair.public_key().clone()],
+        ));
+        self.client.submit_blocking(register_me)?;
+
+        let burn_permission: PermissionToken = CanBurnUserAssets::new(asset_id.clone()).into();
+        let grant_burn_permission_to_alice =
+            GrantBox::new(burn_permission, alice_id.clone()).into();
+        let transfer_permission: PermissionToken =
+            CanTransferUserAssets::new(asset_id.clone()).into();
+        let grant_transfer_permission_to_alice =
+            GrantBox::new(transfer_permission, alice_id).into();
+        let grant_tx = Transaction::new(
+            account_id,
+            Executable::Instructions(vec![
+                grant_burn_permission_to_alice,
+                grant_transfer_permission_to_alice,
+            ]),
+            100_000,
+        )
+        .sign(keypair)?;
+        self.client.submit_transaction(grant_tx)?;
```

Comment:
What is your idea for not using `submit_transaction_blocking()` here?

Reply:
Also, now that every unit creates 3 blocks in the ready phase, please modify this line:

https://github.com/hyperledger/iroha/blob/64d09937ab48bb071284270bd205fd936a3a917a/client/benches/tps/lib.rs#L81

I wish I had made a better way to maintain this consistency